components/ide/jetbrains/image/status/main.go

```diff
@@ -5,75 +5,50 @@
 package main
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net/http"
+	"net/url"
 	"os"
-	"time"
+
+	supervisor "github.com/gitpod-io/gitpod/supervisor/api"
+	"golang.org/x/xerrors"
+	"google.golang.org/grpc"
 )
 
-// proxy for the Code With Me status endpoints that transforms it into the supervisor status format.
 func main() {
 	if len(os.Args) < 2 {
 		fmt.Printf("Usage: %s <port> [<link label>]\n", os.Args[0])
 		os.Exit(1)
 	}
 	port := os.Args[1]
-	label := "Open JetBrains IDE"
+
+	label := "Open in Jetbrain Gateway"
 	if len(os.Args) > 2 {
 		label = os.Args[2]
 	}
 
-	errlog := log.New(os.Stderr, "JetBrains IDE status: ", log.LstdFlags)
+	errlog := log.New(os.Stderr, "Jetbrains IDE status: ", log.LstdFlags)
 
 	http.HandleFunc("/status", func(w http.ResponseWriter, r *http.Request) {
-		var (
```

Comment:
Changes to status endpoint should be part of this PR. We will do it differently in the PR on top.

Reply:
1. Scope of this PR is not to change how JetBrains integration work.
2. This PR breaks JB heartbeating by changes in status endpoint.
3. We will release JB Gateway plugin which will enable special gitpod gateway link and do a minimal change to status endpoint to enable it like here: https://github.com/gitpod-io/gitpod/blob/1d7c3cf1e506ecea7e094fd11e847939fbf21836/components/ide/jetbrains/image/status/main.go#L94

For now, document how to construct a link manually to try it.

Reply:
I meant should **NOT** be in the original comment.